python/build.py

```diff
@@ -49,10 +49,6 @@ def get_python_executable():
         print('Only the wheel with clang will be released to PyPI.')
         sys.exit(-1)
 
-    if not gpu:
-        print('Linux release must ship with the CUDA backend.')
-        sys.exit(-1)
-
```

Comment:
Just to confirm: @yuanming-hu Does this have fatal drawback?
I just want to run `ti dist` on my non-CUDA laptop.

Reply:
Merging this given that v0.6.19 is already released, ff2 revert me if you don't like this.